portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewFields.testcase

```diff
@@ -1499,4 +1813,95 @@ definition {
 		}
 	}
 
+	@description = "LPS-187522. Allow administrators to configure client extension filters on dataset views."
+	@priority = 5
+	test ValidationTestforCreationofClientExtensionFrontendDataSetFilterwithFieldinUs {
+		task ("Given a new Client Extension Frontend Data Set filter") {
+			ClientExtensionGeneral.goToRemoteAppsPortlet();
+
+			ClientExtensionGeneral.addFrontendDataSetFilter(
+				description = "New Client Extension Frontend Data Set Filter",
+				javaScriptURL = "https://liferay.github.io/liferay-frontend-projects/date-filter.js",
+				name = "Client Extension Filter Test");
+		}
+
+		task ("And a new data set entry") {
+			DataSetAdmin.goToDataSetAdminPage();
+
+			DataSetAdmin.createDataSetEntryViaAPI(
+				dataSetName = "Data Set Test",
+				restApplication = "/data-set-manager/fields",
+				restEndpoint = "/",
+				restSchema = "FDSField");
+		}
+
+		task ("And a new view") {
+			DataSetAdmin.createFDSViewViaAPI(
+				dataSetName = "Data Set Test",
+				key_dataSetViewNameList = "View Test");
+		}
+
+		task ("When the user goes to the Filter tab") {
+			DataSetAdmin.goToSpecificViewPage(
+				dataSetName = "Data Set Test",
+				key_viewName = "View Test");
+
+			DataSetAdmin.goToTab(tabName = "Filters");
+		}
+
+		task ("And creates a new client extension filter") {
+			LexiconEntry.gotoAdd();
+
+			MenuItem.click(menuItem = "Client Extension");
+
+			Type(
+				key_text = "Name",
+				locator1 = "TextInput#ANY",
+				value1 = "Liferay Sample Data Set Filter");
+
+			Click(
+				key_indexNumber = 1,
+				key_text = "Select",
+				locator1 = "Button#ANY_N");
+
+			MenuItem.click(menuItem = "name");
+
+			Click(
+				key_indexNumber = 1,
+				key_text = "Select",
+				locator1 = "Button#ANY_N");
+
+			MenuItem.click(menuItem = "Client Extension Filter Test");
+
+			PortletEntry.save();
+
+			LexiconEntry.gotoAdd();
+
+			MenuItem.click(menuItem = "Client Extension");
+
+			Type(
+				key_text = "Name",
+				locator1 = "TextInput#ANY",
+				value1 = "Liferay Sample Data Set Filter");
+
+			Click(
+				key_indexNumber = 1,
+				key_text = "Select",
+				locator1 = "Button#ANY_N");
+
+			MenuItem.click(menuItem = "name");
+
+			Click(
+				key_indexNumber = 1,
+				key_text = "Select",
+				locator1 = "Button#ANY_N");
+
+			MenuItem.click(menuItem = "Client Extension Filter Test");
+		}
+
+		task ("And saves the new filter") {
+			PortletEntry.save();
+		}
+	}
```

Comment:
Where is the rest of the test? It is supposed that you have to create another client extension filter and check the IN USE label and that you cannot create the filter.